packages/statemanager/test/statefulVerkleStateManager.spec.ts

```diff
@@ -40,7 +46,12 @@ describe('Verkle Tree API tests', () => {
 
   it('should return same stateRoot when putting and then deleting account', async () => {
     const trie = await createVerkleTree()
-    const sm = new StatefulVerkleStateManager({ trie, verkleCrypto })
+    const common = new Common({
+      chain: Mainnet,
+      eips: [6800],
+      customCrypto: { verkleCrypto },
```

Comment:
I have a suggestion, can we do the check for the existence of `verkleCrypto` in `Common` inside common? So, if 6800 is active, or it is activated, check if `verkleCrypto` is defined?

Not sure here though, since this would need a check for all cases when 6800 gets active (so when `new Common` constructing it and `common.setActiveEIPs`)

Reply:
Yeah, I was initially going with that, but I realized that:
- We don't seem to do that for any other EIPs (the constructor logic is quite minimal)
- There are situations where we will initially activate the EIP, but only provide verkleCrypto later on by manually assigning `common.customCrypto.verkleCrypto = verkleCrypto` (e.g. b/c of asynchrony considerations)

So I decided do not do that. We could perhaps think of a "validateActiveEIPs" or something like that where we make sure that some EIPs have what they need in order to work. In the context of verkle though (at least with the way we've structured this) I decided to just validate upon usage.